.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,33 @@
+name: prerelease
+
+on: 
+  workflow_dispatch:
+    inputs:
+      prerelease_stage:
+        description: 'alpha or beta'
+        required: true
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Read .nvmrc
+        run: echo "NODE_VERSION=$(cat .nvmrc)" >> $GITHUB_OUTPUT
+        id: nvmrc
+
+      - name: setup node ${{ steps.nvmrc.outputs.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: '${{ steps.nvmrc.outputs.NODE_VERSION }}'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: install dependencies
+        run: yarn
+
+      - name: release npm packages
+        run: yarn run testdrive -- --dist-tag ${{ github.event.inputs.prerelease_stage }}
```

Comment:
When will this be changed to `yarn run release`?

Reply:
I kept it in case reviewers want to test run the workflow. I can change it to release now.